.travis.yml

```diff
@@ -7,8 +7,6 @@ matrix:
     - env: TOXENV=pep8
     - env: TOXENV=py3pep8
     - env: TOXENV=packaging
-    - python: 2.6 # these are just to make travis's UI a bit prettier
```

Comment:
The comment isn't true anymore either. Later versions of Python need it in order to download the Python version on demand.